### PR TITLE
add bpu struct for core_starmc1.h

### PR DESCRIPTION
**Pull request by GuangMing34**

### CODE REVIEW

**Comment by JonatanAntoni** [comment]
@AlbertHuang-CPU, may I ask you to review this contribution, please?

File: CMSIS/Core/Include/core_starmc1.h [hunk 1]
```diff
@@ -1354,6 +1354,62 @@ typedef struct
...
+
+#define BPU_CTRL_KEY_Pos               1U                                      /*!< BPU CTRL: KEY Position */
+#define BPU_CTRL_KEY_Msk               (0x1UL << BPU_CTRL_KEY_Pos)             /*!< BPU CTRL: KEY Mask */
+
+#define BPU_CTRL_ENABLE_Pos            0U                                      /*!< BPU CTRL: ENABLE Position */
+#define BPU_CTRL_ENABLE_Msk            (0x1UL << BPU_CTRL_ENABLE_Pos)          /*!< BPU CTRL: ENABLE Mask */
+
+/** \brief BPU Comparator Register Definitions */
+#define BPU_COMP_BPADDR_Pos            1U                                      /*!< BPU COMP: BPADDR Position */
+#define BPU_COMP_BPADDR_Msk            (0xFFFFFFFEUL << BPU_COMP_BPADDR_Pos)   /*!< BPU COMP: BPADDR Mask */
```
Comment (AlbertHuang-CPU) [inline CMSIS/Core/Include/core_starmc1.h:1406]: The "0xFFFFFFFEUL" shall be "0x7FFFFFFF".

**Review by AlbertHuang-CPU — CHANGES_REQUESTED** [review]

File: CMSIS/Core/Include/core_starmc1.h (hunk 1 above)
Comment (JonatanAntoni) [inline CMSIS/Core/Include/core_starmc1.h:1406]:
```suggestion
#define BPU_COMP_BPADDR_Msk            (0x7FFFFFFFUL << BPU_COMP_BPADDR_Pos)   /*!< BPU COMP: BPADDR Mask */
```

**Comment by JonatanAntoni** [comment]
@GuangMing34, are you still working on this contribution? Could you look into the review comments added by @AlbertHuang-CPU, please?

**Comment by GuangMing34** [comment]
Hi @JonatanAntoni , I saw you already force pushed a new commit to fix it, I'm not familiar with pull request, could you tell me what I should do next? Thanks~

**Comment by JonatanAntoni** [comment]
@GuangMing34, just double check what I pushed. I just accepted the changes requested by @AlbertHuang-CPU. If this is as expected for you I am fine with merging this PR.

**Comment by GuangMing34** [comment]
@JonatanAntoni, already checked, it's ok. Thanks~

**Review by JonatanAntoni — APPROVED** [review]

**Pull request merged**